todo_app.js

```diff
@@ -29,25 +25,41 @@
                 event.target.parentNode.style.textDecoration = 'none';
```

Comment:
If you place `checker` function inside `initApp` method, it won't be visible for other methods,